### DISCUSSION TITLE
Basic manual

### DISCUSSION THREAD

**Discussion by jaygoldie** [dannagle/PacketSender#264] [General]
Hi

I'm new to this forum as well as networking. We have a piece of hardware that I would like to send a packet to and then in turn read the result that comes back from the hardware.

The hardware will be connected directly to my laptop via the RJ-45 connector. 

So my question is, do you have a basic starter guide/manual for Packetsender so I can learn how to send a packet from my laptops RJ-45 port?

Many thanks,
Jason

**Comment by dannagle** [comment]
I don't have anything basic. What I have is at https://packetsender.com/documentation

Though it doesn't help you right now, I am happy to say that the documentation is getting reworked for the next release. I hope to have quick start guides and other tutorials to follow.
